[CI] WatcherYamlRestIT test {p0=mustache/10_webhook/Test webhook action with mustache integration} failing

**Build Scans:**
- [elasticsearch-intake #32459 / part2](https://gradle-enterprise.elastic.co/s/en5xwmy7qti5c)
- [elasticsearch-periodic #11581 / openjdk25_checkpart2_java-matrix](https://gradle-enterprise.elastic.co/s/xmwkfmbdoiu2y)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:watcher:yamlRestTest" --tests "org.elasticsearch.smoketest.WatcherYamlRestIT.test {p0=mustache/10_webhook/Test webhook action with mustache integration}" -Dtests.seed=E48C0A1EA777C8C9 -Dtests.locale=lu-CD -Dtests.timezone=Europe/Ljubljana -Druntime.java=25
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.smoketest.WatcherYamlRestIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dmustache%2F10_webhook%2FTest%20webhook%20action%20with%20mustache%20integration%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: 
Expected: "executed"
     but: was "failed"
```

**Issue Reasons:**
- [main] 2 failures in test test {p0=mustache/10_webhook/Test webhook action with mustache integration} (0.5% fail rate in 393 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -97,9 +97,6 @@ tests:
 - class: org.elasticsearch.upgrades.UpgradeClusterClientYamlTestSuiteIT
   method: test {p0=mixed_cluster/90_ml_data_frame_analytics_crud/Start and stop old regression job}
   issue: https://github.com/elastic/elasticsearch/issues/139654
-- class: org.elasticsearch.smoketest.WatcherYamlRestIT
-  method: test {p0=mustache/10_webhook/Test webhook action with mustache integration}
-  issue: https://github.com/elastic/elasticsearch/issues/139663
 - class: org.elasticsearch.xpack.ml.integration.DatafeedCcsIT
   method: testDatafeedWithCcsRemoteUnavailable
   issue: https://github.com/elastic/elasticsearch/issues/140612

--- a/x-pack/plugin/watcher/src/yamlRestTest/resources/rest-api-spec/test/mustache/10_webhook.yml
+++ b/x-pack/plugin/watcher/src/yamlRestTest/resources/rest-api-spec/test/mustache/10_webhook.yml
@@ -20,7 +20,8 @@
         body:
           pipeline:
             description: _description
-            processors: [ grok: { field: host, patterns : ["(?:\\[(?<hostname>[^\\]]+)\\]|(?<hostname>%{IPORHOST})):(?<port>\\d+)"]} ]
+            processors: [ grok: { field: host, patterns: [ "(?:\\[(?<hostname>[^\\]]+)\\]|(?<hostname>%{IPORHOST})):(?<port>\\d+)" ] },
+                          convert: { field: port, type: integer } ]
           docs: [ { _index: index, _id: id, _source: { host: $host } } ]
   - set: { docs.0.doc._source.hostname: hostname }
   - set: { docs.0.doc._source.port: port }
@@ -28,6 +29,7 @@
   - do:
       watcher.put_watch:
         id: "test_watch"
+        active: false
         body:
           trigger:
             schedule:

